build/scripts/run-shellcheck.sh

```diff
@@ -12,8 +12,14 @@ is_bash() {
     return 1
 }
 
+is_excluded_from_check() {
```

Comment:
Hmmmm .... you're skipping shellcheck here? I think instead we need to fix the static analysis failures.

Reply:
Hmmm... I don't think we want to check and fix potential failures from 3rd party projects - this excludes spdk folder from the check.

Reply:
I guess someone should file an issue with SPDK to fix their static analysis failures in their bash scripts then. :)